addons/mail/models/ir_config_parameter.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models
+
+
+class IrConfigParameter(models.Model):
+    _inherit = 'ir.config_parameter'
+
+    @api.model_create_multi
+    def create(self, vals_list):
```

Comment:
Well I am not sure for this one. Why not simply raising a ValueError or ValidationError so that people aren't surprised that their configuration has another value than what they typed ? That way they will have to update it correctly (or change the conflicting alias). What do you think ?

Reply:
Your suggestion seems nice. We thought to make the mechanism consistent with mail alias as it's not mentioned on the spec what actually should be done. The original chatter msg by Baptiste Vergote seems to state that we may want to automatically rename the aliases / ICP in such cases, but it's not clear anyway :) 

May be you can discuss with PO and let us know.. Thanks :)